config/foreman-proxy-content.migrations/200630000120-disable-pulp-2-el8.rb

```diff
@@ -0,0 +1,4 @@
+if facts[:os][:release][:major] == '8' && facts[:os][:family] == 'RedHat'
```

Comment:
It feels a bit odd to read it this way because I'm so used to first checking the family and then the specific release. Doesn't really matter, just a habit I've developed. Probably because in Puppet modules you often have logic blocks for a family and nested a block for specific releases.